### DISCUSSION TITLE
Add CI

### DISCUSSION THREAD

**Discussion by Lamparter** [IviriusCommunity/Rebound#22] [Ideas]
We should add a CI integration via GitHub Actions to automatically build and test the apps. This would allow for easier generation of assets for releases.

**Comment by errortek** [comment]
> We should add a CI integration via GitHub Actions to automatically build and test the apps. This would allow for easier generation of assets for releases.

Agreed. It would be a lot easier to release releases if apps were built automatically upon every commit, or even manually (when a release is deemed to be ready), as that would still bring a benefit.

**Reply by IviriusMain** [reply]
Well what about app packages?

**Reply by Lamparter** [reply]
Yes you're right. I'll get started on this tomorrow morning!

**Reply by Lamparter** [reply]
> Well what about app packages?

Wdym? Let's talk on discord.

**Reply by errortek** [reply]
> Yes you're right. I'll get started on this tomorrow morning!

Great! I think I'll work on SysInfo tomorrow, depending on if I have time...

**Comment by Lamparter** [comment]
🤔 It doesn't work...
```
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(5034,5): error MSB3030: Could not copy the file "C:\Users\runneradmin\.nuget\packages\microsoft.web.webview2\1.0.2646-prerelease\build\..\\runtimes\win-anycpu\native_uap\Microsoft.Web.WebView2.Core.dll" because it was not found. [D:\a\Rebound\Rebound\Rebound\Rebound.csproj]
```

**Comment by IviriusMain** [comment]
I have something in mind for this now: if we port every app to UWP XAML islands, we can have everything as packaged and with no sandboxing.

**Comment by IviriusMain** [comment]
Working on this rn and it's complicated
